src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -32,6 +32,7 @@ IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
 
 IqPolicyEvaluation.UnableToCommunicate=Unable to communicate with IQ Server: {0}
```

Comment:
Does the 'UnableToCommunicate' still get used?

Reply:
> Does the 'UnableToCommunicate' still get used?

Removed dfb5120